File: fixtures/fragments/js/imports.js
// Example imports from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

import defaultExport from "module-1";
import * as name from "module-2";
import { export1 } from "module-3";
import { export1 as alias } from "module-4";
import { export1 , export2 } from "module-5";
import { export1 , export2 as alias } from "module-6";
import defaultExport, { export1 } from "module-7";
import defaultExport, * as name from "module-8";
import "module-9";
var promise = import("module-10");

// Node require calls
let defaultExport = require("module-11")
let alias = require("module-12").export1;

// Imports that are not detected

const foo = import("foo" + "bar")

const pkg_var = "foo"
const bar = require(pkg_var)

File: fixtures/fragments/r/imports.R
# Usual calls
library(pkg1)
require(pkg2, verbose = TRUE)

# Using the `character.only` option
library("pkg3", character.only = TRUE, verbose = TRUE)

# A library call inside a function
foo <- function () {
    library(pkg4)
}

# Imports that are not detected

library(paste("some_pkg", "_name"))

pkg_var <- "some_package_name"
library(pkg_var, character.only = TRUE)
